config_example.sh

```diff
@@ -115,6 +115,12 @@
 #
 #export BMC_DRIVER="mixed"
 
+#
+# Set libvirt firmware and BMC bootMode
+# Choose "legacy" (bios), "UEFI", or "UEFISecureBoot"
+# Defaults to legacy for ipv4, UEFI for ipv6
+# export BOOT_MODE="UEFI"
```

Comment:
This should be defaulted to "legacy" based on what we have set as a default for this var in vars.md or the other way around?

Otherwise lgtm.

Reply:
I think it defaults to legacy already for ipv4 as line 121 states @furkatgofurov7 ?
/approve